src/test/features/response/form/models/timeline.ts

```diff
@@ -11,7 +11,7 @@ describe('Timeline', () => {
 
       const actual: TimelineRow[] = (new Timeline()).rows
 
-      expect(actual.length).to.equal(4)
+      expect(actual.length).to.equal(INIT_ROW_COUNT)
```

Comment:
Fix: what if INIT_ROW_COUNT is changed to be something different than 4? Test description is explicit that it should be 4, so that's what test should be checking for. Alternatively, the description needs to be changed.

Reply:
done